### ISSUE TITLE
Lagom bundle causing deprecated crypto warning

### ISSUE THREAD

**Issue by fsat** [typesafehub/sbt-conductr#283]
Lagom bundle is causing deprecated crypto warning due to the `-Dplay.crypto.secret` system property set in bundle start command:

https://github.com/typesafehub/sbt-conductr/blob/efe3fb83d31993c604ab0f0884dbba088dde4331/src/main/scala/com/lightbend/conductr/sbt/LagomBundlePlugin.scala#L157-L159

**Important**: before working on this fix, follow up the following Play issue:

https://github.com/playframework/playframework/issues/8075

Basically the gist is Play should not enforce the crypto secret being mandatory unless it's actually used. In Lagom's case, the crypto secret is not used.

If relying the fix on Playframework's side, it may occur in Play `>= 2.6.8`, so a version detection will be necessary as Play <= `2.6.7` will continue to enforce the crypto secret being mandatory.
